docs/Set-PSReadLineOption.md

```diff
@@ -192,13 +192,25 @@ Accept wildcard characters: False
 
 ### -AddToHistoryHandler
 
-Specifies a ScriptBlock that can be used to control which commands get added to PSReadLine history.
+Specifies a ScriptBlock that can be used to control which commands get added to PSReadLine history,
```

Comment:
> how to add a customer handler that also chains to the default.

Support `MulticastDelegate` handler? `+` and `+=` operators in PowerShell don't support combining two delegates.
I feel this might be an overkill. Let's wait for some feedback to improve it.

Reply:
Not multi-cast, but at least expose a public static method that could be called.

Reply:
Got it. Yes, I can make the `GetDefaultAddToHistoryOption` method public. Let me know if you think the method name should be changed.

Reply:
Updated.